controllers/space/space_controller.go

```diff
@@ -324,8 +324,13 @@ func (r *Reconciler) deleteNSTemplateSetFromCluster(logger logr.Logger, space *t
 	}
 	if util.IsBeingDeleted(nstmplSet) {
 		logger.Info("the NSTemplateSet resource is already being deleted")
+		deletionTimestamp := nstmplSet.GetDeletionTimestamp()
+		if time.Since(deletionTimestamp.Time) > 60*time.Second {
+			return true, fmt.Errorf("NSTemplateSet deletion has not completed in over 1 minute")
```

Comment:
I guess that it should return `false` according to the function comment because the NSTemplateSet still exists in this case?

Reply:
oh yes, of course! 
done in 09d608e